x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/results/AnalyticsResult.java

```diff
@@ -56,6 +59,7 @@
         PARSER.declareObject(optionalConstructorArg(), OutlierDetectionStats.STRICT_PARSER, OUTLIER_DETECTION_STATS);
         PARSER.declareObject(optionalConstructorArg(), ClassificationStats.STRICT_PARSER, CLASSIFICATION_STATS);
         PARSER.declareObject(optionalConstructorArg(), RegressionStats.STRICT_PARSER, REGRESSION_STATS);
+        PARSER.declareObject(optionalConstructorArg(), ModelSize.PARSER, MODEL_SIZE_INFO);
```

Comment:
I wonder if there's a good way to avoid the discrepancy: `ModelSize` VS. `model_size_info`. Could the java side by `ModelSizeInfo`? Or do we rather change the c++ to `model_size`?

Reply:
Sure, I will change the object name :)